tcms/testruns/models.py

```diff
@@ -356,7 +354,7 @@ def _status_to_id(cls, status):
             return None
 
     @classmethod
-    def _get_completed_status_ids(cls):
+    def get_completed_status_ids(cls):
```

Comment:
It is used internally (in the same file), but `pylint` raises an issues because we access a private member of another class (`TestCaseRunStatus._get_completed_status_ids`)

Another solution would be to silence the issue.

Reply:
In this case remove the method and replace the only place it is used by a simple query. Also for the query filter use `TestCaseRunStatus.complete_status_names` instead of enumerating the status names by hand.